templates/partials/split_editor.haml

```diff
@@ -219,6 +219,10 @@
                 %input{ng-change:"updateCategory(rule)",ng-model:"rule.test._base",name:"days",number:"",ng-required:"rule.category._base",type:"text",class:"operand-small"}
                 -trans "days"
               .operation{ng-if:"rule._config.operands == 0"}
+
+              .operation{ng-if:'rule._config.type == "has_intent" || len(listBotsIntents) < 0'}
```

Comment:
This operation is displayed if `listBotsIntents` is less than 0? Shouldn't be `>` and with `and` conditional?